src/background.ts

```diff
@@ -43,9 +43,12 @@ browser.windows.onFocusChanged.addListener(async (windowId) => {
 browser.tabs.onUpdated.addListener(async (tabId, changeInfo) => {
   if (changeInfo.status === 'complete') {
     // Get current tab URL.
-    const [tab] = await browser.tabs.query({ active: true, currentWindow: true });
+    const tabs: browser.Tabs.Tab[] = await browser.tabs.query({
+      active: true,
+      currentWindow: true,
+    });
     // If tab updated is the same as active tab
-    if (tabId == tab.id) {
```

Comment:
What if `tabs` is an empty array?

Reply:
It can be an empty array, that's why i added the Validation [here](https://github.com/wakatime/browser-wakatime/pull/205/files#diff-11db236acd6b02b1229e7d325e9bf8edb521032e6388f68b6a884ec24bbe291fR51)

This is the definition for what tabs is `Gets all tabs that have the specified properties, or all tabs if no properties are specified.`

Reply:
Got it, I'm used to other languages where `tabs[0]` would already throw an error before reaching the `?`.